build.rs

```diff
@@ -58,7 +58,6 @@ fn main() {
         target.contains("pc-windows-gnu") ||
         want_static ||
         target != host ||
-        target.contains("musl")
```

Comment:
This doesn't appear to be build-tested; you've removed this line but not the `||` on the previous line.

Reply:
Oh, I did test it but then submitted the patch via the webUI and apparently messed the editing up. Thanks for noticing.